chapter2/tests/t20/terraform/base/alb.tf

```diff
@@ -0,0 +1,46 @@
+module "alb" {
+  source             = "terraform-aws-modules/alb/aws"
+  version            = "8.7.0"
+  name               = local.labels.alb
+  load_balancer_type = "application"
+  vpc_id             = data.aws_vpc.target.id
+  subnets            = data.aws_subnets.wordpress.ids
+  security_groups    = [module.wordpress_alb_sg.security_group_id]
+  target_groups = [
```

Comment:
> Why do we need listener for `80` port? It shouldn't work separately from 443, it should have redirect to 443 at least

@kseniyashaydurova 
We don't need an 80 listener, we listen on 443 and forward everything to the target group on port 80. I didn't redirect from 80 to 443 as usual only because according to the task conditions, there should be a timeout on port 80